test/Data.purs

```diff
@@ -123,36 +127,28 @@ suite = do
           test (show input) do
             fromData (toData input) `shouldEqual` Just input
       group "BigInt" do
-        let
-          inputs =
-            [ BigInt.fromInt 0
-            , BigInt.fromInt 10000
-            , BigInt.fromInt (-10000)
-            ]
-        for_ inputs \input -> do
-          test (show input) do
-            fromData (toData input) `shouldEqual` Just input
-      test "Array" do
-        let
-          input = [ Just true, Just false, Nothing ]
-        fromData (toData input) `shouldEqual` Just input
-      test "Map" do
-        let
-          input = Map
-            [ BigInt.fromInt 13 /\
-                [ Map
-                    [ BigInt.fromInt 17 /\ false
-                    ]
-                ]
-            ]
-        fromData (toData input) `shouldEqual` Just input
+        test "property test" $ quickCheck \(n :: Int) ->
+          let
+            input = BigInt.fromInt n
+          in
+            fromData (toData input) === Just input
+        test "Array" do
+          let
+            input = [ Just true, Just false, Nothing ]
+          fromData (toData input) `shouldEqual` Just input
+        test "Map" do
```

Comment:
Consider refactoring this term into `plutusDataRoundtrip = fromData (toData input) === Just input` and use it across.